### PR TITLE
Kevin/Remove:unused packages

### PR DESCRIPTION
**Pull request by kevinw-binary**
# Description

Summary

Changes:

-   Remove unused packages

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [x] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

**Review by sara-fs — APPROVED** [review]

**Comment by oskar-binary** [comment]
Shouldn't the package-lock.json file be updated by these changes as well? 🤔

**Comment by kevinw-binary** [comment]
> Shouldn't the package-lock.json file be updated by these changes as well? 🤔

Updated the package json file by running `npm install`

**Comment by oskar-binary** [comment]
> > Shouldn't the package-lock.json file be updated by these changes as well? 🤔
> 
> Updated the package json file by running `npm install`

Thanks 👍

**Review by oskar-binary — APPROVED** [review]

File: gatsby-config.js [hunk 1]
```diff
@@ -77,6 +47,14 @@ module.exports = {
                     '/homepage/**',
                     '/**/homepage',
                     '/**/homepage/**',
+                    'careers/jobs',
```
Comment (mustofa-binary) [inline gatsby-config.js:50]:
```suggestion
                    '/careers/jobs',
```

File: gatsby-config.js [hunk 2]
```diff
@@ -77,6 +47,14 @@ module.exports = {
...
+                    '/careers/jobs/**',
+                    'careers/job',
```
Comment (mustofa-binary) [inline gatsby-config.js:52]:
```suggestion
                    '/careers/job',
```

File: gatsby-config.js [hunk 3]
```diff
@@ -77,6 +47,14 @@ module.exports = {
...
+                    '/careers/job/**',
+                    'careers/teams',
```
Comment (mustofa-binary) [inline gatsby-config.js:54]:
```suggestion
                    '/careers/teams',
```

**Review by mustofa-binary — APPROVED** [review]

**Pull request merged**